packages/vuetify/src/labs/VStepper/VStepperItem.tsx

```diff
@@ -0,0 +1,182 @@
+    'group:selected': (val: { value: boolean }) => true,
+  },
+
+  setup (props, { slots }) {
+    const group = useGroupItem(props, VStepperSymbol, true)
+    const step = computed(() => group?.value.value ?? props.value)
+    const isValid = computed(() => props.rules.every(handler => handler() === true))
+    const canEdit = computed(() => !props.disabled && props.editable)
+    const hasError = computed(() => props.error || !isValid.value)
+    const hasCompleted = computed(() => props.complete || (props.rules.length > 0 && isValid.value))
+    const icon = computed(() => {
+      if (hasError.value) return props.errorIcon
+      if (hasCompleted.value) return props.completeIcon
+      if (props.editable) return props.editIcon
+
+      return props.icon
+    })
+    const slotProps = computed(() => ({
+      canEdit: canEdit.value,
+      hasError: hasError.value,
+      hasCompleted: hasCompleted.value,
+      title: props.title,
+      subtitle: props.subtitle,
+      step: step.value,
+    }))
+
+    useRender(() => {
+      const hasColor = (
+        !group ||
+        group.isSelected.value ||
+        hasCompleted.value ||
+        canEdit.value
+      ) && (
+        !hasError.value &&
+        !props.disabled
+      )
+      const hasTitle = !!(props.title || slots.title)
+      const hasSubtitle = !!(props.subtitle || slots.subtitle)
+
+      function onClick () {
+        group?.toggle()
+      }
+
+      return (
+        <button
+          class={[
+            'v-stepper-item',
+            {
+              'v-stepper-item--complete': hasCompleted.value,
+              'v-stepper-item--disabled': props.disabled,
+              'v-stepper-item--error': hasError.value,
+            },
+            group?.selectedClass.value,
+          ]}
+          disabled={ !props.editable }
+          v-ripple={[
+            props.ripple && props.editable,
+            null,
+            null,
+          ]}
+          onClick={ onClick }
+        >
+          <VAvatar
+            key="stepper-avatar"
+            class="v-stepper-item__avatar"
+            color={ hasColor ? props.color : undefined }
+            size={ 24 }
+          >
+            { slots.icon?.(slotProps.value) ?? (
+              icon.value ? (
+                <VIcon icon={ icon.value }></VIcon>
+              ) : step.value
+            )}
+          </VAvatar>
+
+          <div class="v-stepper-item__content">
+            { hasTitle && (
+              <div
+                key="title"
+                class="v-stepper-item__title"
+              >
+                { slots.title?.(slotProps.value) ?? props.title }
+              </div>
+            )}
+
+            { hasSubtitle && (
+              <div
+                key="subtitle"
+                class="v-stepper-item__subtitle"
+              >
+                { slots.subtitle?.(slotProps.value) ?? props.subtitle }
+              </div>
+            )}
+
+            { slots.default?.(slotProps.value) }
```

Comment:
I think it might be unexpected that `item` slot does not override the content but simply appends at the end.